spec.bs

```diff
@@ -1842,6 +1845,34 @@ Each {{Window}} object has an associated <dfn for=Window>fence</dfn>, which is a
   </wpt>
 </div>
 
+<h3 id=document-extension>Extensions to the {{Document}} interface</h3>
```

Comment:
"Interface" is specifically used when we're modifying or extending an IDL interface, but the stuff in this section is more about internal infra / supporting concepts that are scoped to a Document. Given that, I'd either:
 1. Rename this section to `{{Document}} supporting concepts`
 2. Move its contents elsewhere we define things on the "Document" object (assuming we do that elsewhere in the spec).

I'll let you make the call.

Reply:
I don't think we modify `{{Document}}` in the same way elsewhere in the spec, so I'll rename the section.